reference/release-notes/1.23.1.md

```diff
@@ -0,0 +1,118 @@
+---
+orphan: true
+---
+# 1.23.1
+
+These release notes cover new features and changes in Anbox Cloud 1.23.1.
+
+Anbox Cloud 1.23.1 is a patch release. To understand minor and patch releases, see {ref}`ref-release-notes`.
+
+Please see {ref}`ref-component-versions` for a list of updated components.
+
+## Requirements
+
+See the {ref}`ref-requirements` for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Anbox Cloud Appliance
+
+* You can now set the following appliance network configuration using `anbox-cloud-appliance config set`:<!--AC-2732-->
+  - The network's public IP address (`network.public_address`)
+  - The network's DNS name (`network.location`)<!--AC-2727-->.
+* You can now configure the [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for those API endpoints that the appliance reverse proxy passes to the Anbox Stream Gateway. You can set the following using the `anbox-cloud-appliance config set` command:<!--AC-2732-->
+  - The [HTTP origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) (`core.https_allowed_origin`)
+  - List of allowed HTTP headers (`core.https_allowed_headers`)
+  - List of allowed HTTP methods (`core.https_allowed_methods`)
+* You can skip setting up coturn when initialising the appliance and can configure a custom STUN server.<!--AC-2719-->
+* The [`prepare-node-script`](prepare-node-script.md) command is extended to support systems with an already installed NVIDIA driver. <!--AC-2714-->
+* You can now set up a custom identity provider for the dashboard by configuring an OpenID Connect provider through the preseed configuration when initialising the appliance.
+
+### Streaming
+
+* With the 1.23.0 release, the functionalities of the *Instances* page and the *Sessions* page on the web dashboard were merged. For debugging purposes, viewing a list of available sessions can be really useful. With the 1.23.1 release, you can list all available sessions using the `anbox-stream-gateway sessions list` command.<!--AC-2662-->
+* You can dynamically change the display density when joining an existing streaming session.<!--AC-2673-->
+
+### VHAL
+
+* Starting with 1.23.0, the VNDK version required for building a custom VHAL is 34.<!--AC-2720-->
+
+### Android vendor image
+
+* WiFi support for Android 12 vendor images.<!--AC-2707-->
```

Comment:
We actually added Wi-Fi support for all Android versions.
```suggestion
### Android images

* WiFi support for all Android versions.<!--AC-2707-->
```
This was a known issue for 1.23.0